tests/functional/testSubscription.js

```diff
@@ -19,14 +38,195 @@ describe('Subscription view', function() {
+        status: 200,
+        body: null,
+        callback: (req) => this.ctx.fxaTotpCallback(req)
+      },
+      '/v1/account/destroy': {
+        status: 200,
+        body: null,
+        callback: (req) => this.ctx.fxaDestroyCallback(req)
+      },
+    },
+    DELETEs: {},
+  };
 
   beforeEach(async () => {
     if (!(await vpn.isFeatureFlippedOn('subscriptionManagement'))) {
       await vpn.flipFeatureOn('subscriptionManagement');
     }
+    if ((await vpn.isFeatureFlippedOn('accountDeletion'))) {
+      await vpn.flipFeatureOff('accountDeletion');
+    }
+  });
+
+  it('Authentication needed - sample', async () => {
+    this.ctx.fxaLoginCallback = (req) => {
+      this.ctx.fxaOverrideEndpoints.POSTs['/v1/account/login'].body = {
+        sessionToken: 'session',
+        verified: true,
+        verificationMethod: ''
+      };
+      this.ctx.fxaOverrideEndpoints.POSTs['/v1/account/login'].status = 200;
+    };
+    this.ctx.guardianSubscriptionDetailsCallback = req => {
+      this.ctx.guardianOverrideEndpoints.GETs['/api/v1/vpn/subscriptionDetails']
+          .status = 401;
+      this.ctx.guardianOverrideEndpoints.GETs['/api/v1/vpn/subscriptionDetails']
+          .body = {}
+    };
+
+    await vpn.waitForElement('settingsButton');
+    await vpn.clickOnElement('settingsButton');
+
+    await vpn.waitForElement('settingsUserProfile');
+    await vpn.waitForElementProperty('settingsUserProfile', 'visible', 'true');
+
+    await vpn.waitForElement('settingsUserProfile-displayName');
+    await vpn.waitForElementProperty(
+        'settingsUserProfile-displayName', 'visible', 'true');
+    await vpn.waitForElementProperty(
+        'settingsUserProfile-displayName', 'text', 'Test test');
+
+    await vpn.waitForElement('settingsUserProfile-emailAddress');
+    await vpn.waitForElementProperty(
+        'settingsUserProfile-emailAddress', 'visible', 'true');
+    await vpn.waitForElementProperty(
+        'settingsUserProfile-emailAddress', 'text', 'test@mozilla.com');
+
+    await vpn.waitForElement('settingsUserProfile-manageAccountButton');
+    await vpn.waitForElementProperty(
+        'settingsUserProfile-manageAccountButton', 'visible', 'true');
+    await vpn.clickOnElement('settingsUserProfile-manageAccountButton');
+
+    await vpn.waitForElement('authSignIn-passwordInput');
+    await vpn.waitForElementProperty(
+        'authSignIn-passwordInput', 'visible', 'true');
+    await vpn.setElementProperty(
+        'authSignIn-passwordInput', 'text', 's', 'P4ass0rd!!');
+
+    await vpn.waitForElementProperty('authSignIn-button', 'enabled', 'true');
+
+    this.ctx.guardianSubscriptionDetailsCallback = req => {
+      this.ctx.guardianOverrideEndpoints.GETs['/api/v1/vpn/subscriptionDetails']
+          .status = 200;
+      this.ctx.guardianOverrideEndpoints.GETs['/api/v1/vpn/subscriptionDetails']
+          .body = SUBSCRIPTION_DETAILS;
+    };
+
+    await vpn.clickOnElement('authSignIn-button');
```

Comment:
The button is not clicked if I’m running the test locally. Adding `await vpn.wait();` before this line fixes the issue. Which is weird, because we are already waiting for the button to enabled above.